tests/ActionBuilder/Cart/SetLineItemExternalPriceTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BestIt\CommercetoolsODM\Tests\ActionBuilder\Cart;
+
+use BestIt\CommercetoolsODM\ActionBuilder\Cart\SetLineItemExternalPrice;
+use BestIt\CommercetoolsODM\Mapping\ClassMetadataInterface;
+use Commercetools\Core\Model\Cart\Cart;
+use Commercetools\Core\Model\Cart\LineItem;
+use Commercetools\Core\Model\Cart\LineItemCollection;
+use Commercetools\Core\Model\Common\Money;
+use Commercetools\Core\Model\Common\Price;
+use Exception;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Tests for the set line item external price action of cart update request.
+ *
+ * @author Michel Chowanski <michel.chowanski@bestit-online.de>
+ * @package BestIt\CommercetoolsODM\Tests\ActionBuilder\Cart
+ */
+class SetLineItemExternalPriceTest extends TestCase
+{
+    /**
+     * Test that the created actions contains the correct external price.
+     *
+     * @return void
+     *
+     * @throws Exception
+     */
+    public function testThatActionsContainsCorrectExternalPrice()
```

Comment:
Since when did you start to use "that" behat style? There is no Method "thatActionsContainsCorrectExternalPrice".